### ISSUE TITLE
Windows x86 client absent

### ISSUE THREAD

**Issue by bprykhodchenko** [google/grr#980]
**Environment**
The GRR is installed on a VM running on ESXi on-prem. The VM runs Ubuntu 18.04 and GRR was installed from DEB (using the official documentation)
GRR Version is 3.4.6.0
Ubuntu 18.04
Windows 2003

**Describe the issue**
1. Why is there no x86 client present in the latest GRR Server?
2. Do you have a page with supported OS versions/distributions? Because I didn't find it on the official website...

The issue is that we have Windows Server 2003 and we have a limited visibility into it. A lot of modern security preventive and detective products do not support such old systems. So I wanted to do some investigation on it using GRR, but nope, x86 client is not available and amd64 is not compatible with such archaic x86 systems....

**Comment by mbushkov** [comment]
Hey, thanks for the question. Please see a reply to a similar question here: https://github.com/google/grr/issues/774#issuecomment-628721889

TLDR is: we no longer provide 32 bit installers out of the box, but if you can build Python 3.6 for that platform and if all the dependent Python packages can be built as well, then it should be possible to create the custom x86 GRR client installer.
